### PR TITLE
Update secp256k1 zkp

### PR DESCRIPTION
**Pull request by altafan**
This updates the dependency `@vulpemventures/secp256k1-zkp` to its latest version `v3.0.0`, which is essentially an extension of the `TinySecp256k1Interface` implementation of `bitcoinjs/tiny-secp`.

Our implementation and interface include also zkp methods but can still be used for signing, giving devs the chance to use it in other projects as an alternative to tiny-secp.

With this, liquidjs doesn't import any `TinySecpInterface` defined in other bitcoinjs projects (ecpair), instead, it defines the `Secp256k1Interface` interface in a new ad hoc file _secp256k1-zkp.ts_.

Please @louisinger @tiero review this.

### CODE REVIEW

File: ts_src/psetv2/blinder.ts [hunk 1]
```diff
@@ -164,10 +164,11 @@ export class Blinder {
           value,
           targetOutput.asset!,
           valueCommitment,
+          assetCommitment,
```
Comment (tiero) [inline ts_src/psetv2/blinder.ts:167]: the order of parameters changed, looks like a breaking change to me. if was intentional I suggest tagging v3.x.x so we make evident the signature change

File: src/bip341.d.ts [hunk 2]
```diff
@@ -1,23 +1,16 @@
 /// <reference types="node" />
-import { TinySecp256k1Interface as ECPairSecp256k1Interface } from 'ecpair';
```
Comment (louisinger) [inline src/bip341.d.ts:2]: This should be reverted, liquidjs must not depend on our implementation.

File: src/confidential.d.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /// <reference types="node" />
 import { Output } from './transaction';
+import type { ZKP as ZKPInterface } from '@vulpemventures/secp256k1-zkp';
```
Comment (louisinger) [inline src/confidential.d.ts]: We maybe should revert this if we want to be 100% consistent with bitcoinjs libs. Moreover may be useful to have a global Secp interface defined in liquidjs while using it?

File: ts_src/psetv2/pset.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
-import { ECPairFactory, TinySecp256k1Interface } from 'ecpair';
```
Comment (louisinger) [inline ts_src/psetv2/pset.ts:1]: Not needed I think

**Review by louisinger — CHANGES_REQUESTED** [review]
Any reason to do not support bip32 interfaces & slip77 ones ? If the idea is to fully drop tinysecp support in upper projects they may need them also

File: src/confidential.d.ts (hunk 3 above)
Comment (tiero) [reply src/confidential.d.ts]: What if the interface is stored in LiquidJS? so we leave `@vulpemventures/secp256k1-zkp` and upcoming `@vulpemventures/react-native-secp256k1-zkp` independent to each other

File: src/bip341.d.ts (hunk 2 above)
Comment (altafan) [reply src/bip341.d.ts:2]: I'm going to add a secp interface to liquidjs. You're correct we shouldn't depend on our impl, but we don't want to depend on other projects either for this interface

File: src/confidential.d.ts (hunk 3 above)
Comment (altafan) [reply src/confidential.d.ts]: Yeah originally I did this, i'm going to restore this interface.

**Review by louisinger — COMMENTED** [review]
We should also add export statement for our Secp256k1 interface in `index.ts`

**Review by louisinger — APPROVED** [review]

**Pull request merged**